explainer.md

```diff
@@ -0,0 +1,107 @@
+# Screen Wake Lock API
+
+## Introduction
+The **Screen Wake Lock API** allows web applications to prevent a device's
+screen from dimming and locking.
+
+See also:
+* [The current specification](https://w3c.github.io/screen-wake-lock/)
+* [web.dev article](https://web.dev/wakelock/) on the Screen Wake Lock API
+
+## Why?
+To avoid draining the battery, most devices quickly go to sleep when left idle.
+While this is fine most of the time, some applications need to keep the screen
+awake to complete their work. For example:
+
+* A recipe app that keeps the screen on while you bake a cake or cook dinner.
+* A boarding pass or ticket app that keeps the screen on until the barcode has
+  been scanned.
+* A kiosk-style app that keeps the screen on continuously.
```

Comment:
I don't think this is a valid use case for the API. 

```suggestion
```

This is best handled at the OS level.

Reply:
Actually, at Maersk they are using it for exactly this reason, for dashboards etc. You could say that they should control that at OS level, but sometimes it is hard to educate users to make those changes or even get the admin rights. Also you might not want people to always turn off the screen sleep, because it should only work when the app is running.

Reply:
That use case from Maersk sounds reasonable to me.

@marcoscaceres can you expand why it is not valid in your opinion?

Reply:
That's exactly what we use it for (kiosk-style dashboard) - but also considering it for always on monitoring screens/devices on vessels.

Reply:
Ok, we should clarify this a bit then. It sounds like a web apps on a kiosk specific device.

Reply:
I think there is a spectrum of kiosk-style devices. Some hardware is specifically designed for use as a kiosk and may include built-in functionality for keeping the screen on all the time or on a fixed schedule. On the other hand some "kiosks" are more ad-hoc and simply reuse commodity hardware and software. I think it is reasonable for any site that presents dashboards to have the ability to ask the browser to keep the screen on because that is frequently what dashboards are used for.

Reply:
Ok, happy for us to mention this. But just noting that this will likely be controversial and/or put off potential implementers.